lambdas/access_counts/index.py

```diff
@@ -210,8 +210,9 @@ def sql_escape(s):
             eventname,
             bucket,
             lower(CASE
-                WHEN cardinality(parts) > 2 THEN concat(element_at(parts, -2), '.', element_at(parts, -1))
-                WHEN cardinality(parts) = 2 THEN element_at(parts, -1)
+                WHEN cardinality(parts) > 2 AND lower(element_at(parts, -1)) = 'gz'
+                    THEN concat(element_at(parts, -2), '.', element_at(parts, -1))
+                WHEN cardinality(parts) >= 2 THEN element_at(parts, -1)
```

Comment:
Why did you change it to `>=` here?

Reply:
So that when `bucket/folder/foo.baz.png` flunks the line above, it still turns into  png?